reader/components/print/Print.jsx

```diff
@@ -81,7 +84,7 @@ class Print extends React.Component {
     }
 
     this.pagesOverview = viewer.getPagesOverview()
-    return this._renderPages()
+    return this.renderPages()
```

Comment:
You could leave `underscore-dangle` rule for this project. It's up to you.

Reply:
The projects should be unified in code style as much as possible.